setup.py

```diff
@@ -20,6 +20,8 @@
     "pandas==0.25.3",
     "result_caching @ git+https://github.com/mschrimpf/result_caching",
     "jupyter",
+    "matplotlib",
+    "tensorflow==1.15",
```

Comment:
so far, we left the modeling libraries out of the setup.py so that people can set them up with conda, but this is probably for the OST stuff right?

Reply:
Yes but this is a problem for the CI. If the library is not there it will fail, so from now on to pass the test gate on jenkins you will have to add all requried libraries in setup.py. Otherwise they are not installed. And yes I need tensorflow for the OST stuff afaik.